ui/src/features/Editor/components/Canvas/components/Nodes/NoteNode/index.tsx

```diff
@@ -108,7 +110,23 @@ const NoteNode: React.FC<NoteNodeProps> = ({ data, ...props }) => {
           <Note />
           <p>{data.params?.customName ?? data.officialName}</p>
         </div>
-        <textarea
+        <span
```

Comment:
If there is no real reason to make this a span, can you do a div (bg color) + p (obv text stuff) instead? 🙏🏻